target/openjdk-8-stretch/tools-deps/Dockerfile

```diff
@@ -1,17 +1,13 @@
 FROM openjdk:8-stretch
-LABEL maintainer="Paul Lam <paul@quantisan.com> & Wes Morgan <wesmorgan@icloud.com>"
 
 ENV CLOJURE_VERSION=1.10.1.469
 
 WORKDIR /tmp
 
+RUN \
+wget https://download.clojure.org/install/linux-install-$CLOJURE_VERSION.sh && \
+chmod +x linux-install-$CLOJURE_VERSION.sh && \
+./linux-install-$CLOJURE_VERSION.sh && \
+clojure -e "(clojure-version)"
 
-RUN wget https://download.clojure.org/install/linux-install-$CLOJURE_VERSION.sh \
-    && chmod +x linux-install-$CLOJURE_VERSION.sh \
-    && ./linux-install-$CLOJURE_VERSION.sh
-
-RUN clojure -e "(clojure-version)"
-
-# Docker bug makes rlwrap crash w/o short sleep first
-# Bug: https://github.com/moby/moby/issues/28009
 CMD ["sh", "-c", "sleep 1 && exec clj"]
```

Comment:
Is this `rlwrap` thing still relevant?  Did you want to keep the comment here that explains why this includes a `sleep 1` that otherwise looks odd?

Reply:
Yeah I meant to check on that. I’ll do so and clean this up one way or the other.

Reply:
Tested and the bug still exists. I put the comment back in with a note that it still exists as of 2019-10-2 even though the linked issue is closed.